### ISSUE TITLE
regression

### ISSUE THREAD

**Issue by UnixJunkie** [UnixJunkie/orf#2]

**Comment by UnixJunkie** [comment]
done, need to test

**Comment by UnixJunkie** [comment]
I suspect this test set is too hard.
Need to check with sklearn.

**Comment by UnixJunkie** [comment]
done
